script_sklearn_model

```diff
@@ -0,0 +1,48 @@
+import pickle
+import pandas as pd
+from sklearn.linear_model import LinearRegression
+#from google.cloud import storage
+
+"""
```

Comment:
here i have added the code to save the trained models to GCS .If you want to use it just remove the inverted comma and it will  be there to work.